packages/ruby-2.1.8/packaging

```diff
@@ -15,7 +15,7 @@ tar xzf ruby-2.1.8/yaml-${LIBYAML_VERSION}.tar.gz
     cp ${BOSH_COMPILE_TARGET}/architecture-support/config/config.{guess,sub} ./config
   fi
   ./configure --prefix=${BOSH_INSTALL_TARGET}
-  make -j 3   # Use 3 CPUs, which is 1.5 x 2 (the normal number of compile CPUs)
+  make -j $(( $(getconf _NPROCESSORS_ONLN) * 3 / 2 ))
```

Comment:
What about using ```make -j $(getconf _NPROCESSORS_ONLN)``` instead?

It is simpler and might be a little faster:
* https://blogs.gentoo.org/ago/2013/01/14/makeopts-jcore-1-is-not-the-best-optimization/
* https://wiki.gentoo.org/wiki/MAKEOPTS